lib/modules/manager/custom/index.ts

```diff
@@ -1,12 +1,17 @@
-import type { ExtractConfig, PackageFileContent, Result } from '../types';
+import type { ManagerApi } from '../types';
+import customManagers from './api';
 
-// export hollow functions for validation as manager
-export const defaultConfig = {};
-export const supportedDatasources = ['*'];
-export function extractPackageFile(
-  content: string,
-  packageFile: string,
-  config: ExtractConfig
-): Result<PackageFileContent | null> {
-  return { deps: [] };
+const customManagerList = Array.from(customManagers.keys());
+export const getCustomManagerList = (): string[] => customManagerList;
```

Comment:
I think we can simply export the array, the wrapper function don't make the code better or safer